### PR TITLE
add vpg endpoints

### PR DESCRIPTION
**Pull request by ariquinones**
issue #50

### CODE REVIEW

**Comment by JoMas971** [comment]
This PR looks fine to me. Good job man

File: src/sdk/model/vpg/json/vpg-failover-create-request.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Vpg Failover Create Request properties
+ */
+export interface VpgFailoverCreateRequest {
```
Comment (cfsnyder) [inline src/sdk/model/vpg/json/vpg-failover-create-request.ts]: Names of all interfaces in the `json` folder should end in 'Json', and all should be wrapped by a class at the SDK level. Use Nico's pattern for request classes that we talked about as a group.

File: src/sdk/model/vpg/json/vpg-failover-test-alert-request.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Vpg Failover Test Alert Request
+ */
+export interface VpgFailoverTestAlertRequest {
```
Comment (cfsnyder) [inline src/sdk/model/vpg/json/vpg-failover-test-alert-request.ts]: another one that needs to be renamed and wrapped by a class.

File: src/sdk/model/vpg/json/vpg-failover-create-request.ts (hunk 1 above)
Comment (JoMas971) [reply src/sdk/model/vpg/json/vpg-failover-create-request.ts]: Ho yeah. Good catch !

**Comment by cfsnyder** [comment]
thanks @ariquinones !

**Pull request merged**